cmd/tailscaled/tailscaled.go

```diff
@@ -340,9 +340,7 @@ func run() error {
 		sys.InitialConfig = conf
 	}
 
-	netMon, err := netmon.New(func(format string, args ...any) {
-		logf(format, args...)
-	})
+	netMon, err := netmon.New(logf)
```

Comment:
Could you explain why you're removing the logging?

Reply:
I don't remove the logging.
I think `logf` is same as 
```go
func(format string, args ...any) {
	logf(format, args...)
}
```